client/order/index.js

```diff
@@ -1,30 +1,58 @@
 /* global jQuery */
 
-import { __, _n, sprintf } from '@wordpress/i18n';
-import { dateI18n } from '@wordpress/date';
 import ReactDOM from 'react-dom';
+import React from 'react';
+import { __ } from '@wordpress/i18n';
 import { dispatch } from '@wordpress/data';
-import moment from 'moment';
-import { createInterpolateElement } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { getConfig } from 'utils/order';
+import { isAwaitingResponse, isUnderReview } from 'wcpay/disputes/utils';
 import RefundConfirmationModal from './refund-confirm-modal';
 import CancelConfirmationModal from './cancel-confirm-modal';
-import InlineNotice from 'components/inline-notice';
-import { formatExplicitCurrency } from 'utils/currency';
-import { reasons } from 'wcpay/disputes/strings';
-import { getDetailsURL } from 'wcpay/components/details-link';
-import {
-	isAwaitingResponse,
-	isInquiry,
-	isUnderReview,
-} from 'wcpay/disputes/utils';
-import { useCharge } from 'wcpay/data';
-import wcpayTracks from 'tracks';
-import './style.scss';
+import DisputedOrderNoticeHandler from 'components/disputed-order-notice';
```

Comment:
I remember @Jinksi usually would suggest that we prefix the import path with `wcpay/`.

```suggestion
import DisputedOrderNoticeHandler from 'wcpay/components/disputed-order-notice';
```

Reply:
@Jinksi any ideas if there are tools we can use to standardise our imports (order, relative, roots etc)? Would be cool to get a robot on that.

Reply:
PS I like `wcpay/components` as it disambiguates which components we are talking about (vs `@wordpress` and `@woocommerce`).

Reply:
> any ideas if there are tools we can use to standardise our imports

I'm not sure about this one. We could change our config to remove the [`wcpay/` alias](https://github.com/Automattic/woocommerce-payments/blob/4e872a9e1bce8729f2eae97be1c131df555c17a8/webpack/shared.js#L115-L117) or remove each [`client/subdir` alias](https://github.com/Automattic/woocommerce-payments/blob/4e872a9e1bce8729f2eae97be1c131df555c17a8/webpack/shared.js#L114).

> I like wcpay/components as it disambiguates which components we are talking about

There was a discussion in Slack and it seems most were in favour of omitting `wcpay/`, although it was a small sample size: p1696552621397799-slack-C053VQPD74J.

Reply:
🤷🏻 I guess this is pretty minor so let's not worry about it too much